main.js

```diff
@@ -1,163 +1,388 @@
+        name: 'HD TV',
+        brand: 'Toshiba',
+        price: 161,
+        availableSizes: [32],
+        refreshRate: 50,
+        screenType: 'LED',
+        screenQuality: 'Full HD',
+        smartTv: false,
+        options: {
+            wifi: false,
+            speech: false,
+            hdr: true,
+            bluetooth: false,
+            ambiLight: false,
+        },
+        originalStock: 10,
+        sold: 8,
     },
-    originalStock: 10,
-    sold: 8,
-  },
-];
\ No newline at end of file
+];
+/*
+Opdracht 1 - Array Methoden
+Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+    Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+Opdracht 1c: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die over AmbiLight beschikken. Log de uitkomst in de console.
+Opdracht 1d: Schrijf een functie die alle tv's van laagste naar hoogste prijs sorteert. Log de uitkomst in de console.
+*/
+
+//Opdracht 1a:
+//Loop de array langs en return alleen de typenamen van de tv's. Log in console.
+
+const allTvTypes = inventory.map((tvType) => {
+    return tvType.type;
+})
+console.log(allTvTypes);
+
+//Opdracht 1b:
+//Loop de array langs en return alle tv's waarbij 'original stock' en 'sold' gelijk zijn en log in de console.
+
+const soldOut = inventory.filter((invent) => {
+    return invent.sold === invent.originalStock;
+})
+console.log(soldOut);
+
+//Opdracht 1c:
+// loop de array langs en return de tv's met 'ambilight: true' en log in de console.
+
+const ambiLight = inventory.filter((ambi) => {
+    return ambi.options.ambiLight === true;
+})
+console.log(ambiLight);
+
+//Opdracht 1d:
+// Loop de array langs en sorteer de tv's van laag naar hoog in prijs en log in de console.
+
+inventory.sort((a, b) => {
+    return a.price - b.price;
+})
+console.log(inventory);
+
+/*
+Opdracht 2 - Elementen in de DOM plaatsen
+Tip: wanneer we meerdere waardes uit een array willen terugbrengen tot één getal of één string, gebruik je hier gewoon een oude vertrouwde for-loop voor!
+
+Opdracht 2a: Hoeveel tv's zijn er al verkocht? Schrijf een script dat dit berekent. Log de uitkomst in de console.
+Opdracht 2b: Zorg ervoor dat dit aantal in het groen wordt weergegeven op de pagina.
+Opdracht 2c: Hoeveel tv's heeft Tech It Easy ingekocht? Schrijf een script dat dit berekent. Log de uitkomst in de console.
+Opdracht 2d: Zorg ervoor dat dit aantal in het blauw wordt weergegeven op de pagina.
+Opdracht 2e: Geef in het rood weer hoeveel tv's er nog verkocht moeten worden.
+ */
+
+//Opdracht 2a:
+//maak een for-loop waarin het aantal verkochte tv's bij 'inventory.sold' steeds onthouden en opgeteld worden.
+//log het totaal aantal verkochte tv's in de console.
+
+let howManySold = 0
+for (let i = 0; i < inventory.length; i++) {
+    howManySold = howManySold + inventory[i].sold;
```

Comment:
Hier kun je de `+=` gebruiken
```suggestion
    howManySold += inventory[i].sold;
```